README.md

```diff
@@ -1 +1,40 @@
-# North-portifolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portifolio
+
+> A Project detailing about me and my works.
+
+
+## Built With
+
+- HTML 5
+-CSS
+-Linters
+
+## Live Demo (un-available)
+
```

Comment:
- [ ] For the sake of people that don't have enough technical knowledge, I suggest you Kindly add detailed installation guidelines to your README FILE. e.g;

### Installation
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

`git clone https://github.com/rskinnerc/portafolio.git`
Once the files are on your machine, open the portafolio folder in [Visual Studio Code](https://code.visualstudio.com/).

With the files open in Visual Studio Code, press the Go Live button at the bottom of the window to launch the files with [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer).